docs/zh-cn/_summary.md

```diff
@@ -1,133 +1,192 @@
-* [完全菜鸟指南](zh-cn/newbs.md)
+<!--for translators, see first: zh-cn/reference_glossary.md#terms-of-zh-cn-translate -->
+* 新手教程
+  * [介绍](zh-cn/newbs.md)
   * [入门](zh-cn/newbs_getting_started.md)
-  * [构建你的第一个固件](zh-cn/newbs_building_firmware.md)
-  * [刷新固件](zh-cn/newbs_flashing.md)
-  * [测试和调试](zh-cn/newbs_testing_debugging.md)
-  * [Git最佳实践](zh-cn/newbs_git_best_practices.md)
-    * [使用你分叉(fork)的主分支(master)](zh-cn/newbs_git_using_your_master_branch.md)
-    * [解决合并冲突](zh-cn/newbs_git_resolving_merge_conflicts.md)
-    * [重新同步一个分支](zh-cn/newbs_git_resynchronize_a_branch.md)
-  * [学习资源](zh-cn/newbs_learn_more_resources.md)
-
-* [QMK基础](zh-cn/README.md)
-  * [QMK简介](zh-cn/getting_started_introduction.md)
-  * [QMK命令行工具](zh-cn/cli.md)
-  * [QMK命令行工具配置](zh-cn/cli_configuration.md)
-  * [向QMK贡献代码](zh-cn/contributing.md)
-  * [如何使用GitHub](zh-cn/getting_started_github.md)
-  * [获得帮助](zh-cn/getting_started_getting_help.md)
-
-* [非兼容性修改](zh-cn/breaking_changes.md)
-  * [我的PR已经被标记为非兼容性修改](zh-cn/breaking_changes_instructions.md)
-  * [2019年8月30日](zh-cn/ChangeLog/20190830.md)
-
-* [问题与解答](zh-cn/faq.md)
-  * [一般问题](zh-cn/faq_general.md)
-  * [构建/编译](zh-cn/faq_build.md)
-  * [调试/故障排除](zh-cn/faq_debug.md)
-  * [布局](zh-cn/faq_keymap.md)
-  * [Zadig驱动安装](zh-cn/driver_installation_zadig.md)
-
-* 详细指南
-  * [安装构建工具](zh-cn/getting_started_build_tools.md)
-  * [vagrant指南](zh-cn/getting_started_vagrant.md)
-  * [构建/编译指南](zh-cn/getting_started_make_guide.md)
-  * [刷新固件](zh-cn/flashing.md)
-  * [定制功能](zh-cn/custom_quantum_functions.md)
-  * [布局概述](zh-cn/keymap.md)
-
-* [硬件](zh-cn/hardware.md)
-  * [兼容的单片机](zh-cn/compatible_microcontrollers.md)
-  * [AVR处理器](zh-cn/hardware_avr.md)
-  * [驱动](zh-cn/hardware_drivers.md)
-
-* 参考
-  * [键盘指南](zh-cn/hardware_keyboard_guidelines.md)
-  * [配置选项](zh-cn/config_options.md)
-  * [键码](zh-cn/keycodes.md)
-  * [代码书写规范 - C](zh-cn/coding_conventions_c.md)
-  * [代码书写规范 - Python](zh-cn/coding_conventions_python.md)
-  * [文档书写规范](zh-cn/documentation_best_practices.md)
-  * [文档模板](zh-cn/documentation_templates.md)
+  * [构建第一个固件](zh-cn/newbs_building_firmware.md)
+  * [刷写固件](zh-cn/newbs_flashing.md)
+  * [寻求帮助](zh-cn/support.md)
+  * [其它资源](zh-cn/newbs_learn_more_resources.md)
+  * [QMK知识大纲](zh-cn/syllabus.md)
+
+* FAQ
```

Comment:
```suggestion
* FAQ
```

Ditto for all the similar empty lines in this file.

Reply:
we should keep these empty lines, since origin en version have them